common/lib/xmodule/xmodule/graders.py

```diff
@@ -437,7 +438,7 @@ def total_with_drops(breakdown, drop_count):
 
             if not self.hide_average:
                 breakdown.append({'percent': total_percent, 'label': total_label,
-                                  'detail': total_detail, 'category': self.category, 'prominent': True})
+                                  'detail': total_detail, 'category': self.category, 'prominent': True, 'only': True, 'subsection': subsection,})
```

Comment:
what does `'only': True` means?

Reply:
I already added a code comment about that.